packages/components/src/layouts/Navigation/Tab/TabBar/DesktopTabItem.tsx

```
@@ -111,7 +111,7 @@ export function DesktopTabItem(
 
   const intl = useIntl();
   const stackRef = useRef<TamaguiElement>(null);
-  const openActionList = useRef<() => void | undefined>();
+  const openActionList = useRef<() => void | undefined>(undefined);
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)